src/gui/shell.cpp

```diff
@@ -55,13 +56,20 @@ Shell::Shell(NeovimConnector *nvim, QWidget *parent)
 {
 	setAttribute(Qt::WA_KeyCompression, false);
 
+	currentInputMethod = QGuiApplication::inputMethod();
+	currentLocale = currentInputMethod->locale();
```

Comment:
Move this up to the initializer list:
```
  currentInputMethod{ QGuiApplication::inputMethod() };
  currentLocale{ currentInputMethod->locale() };
```

Using that `{}` init syntax helps the compiler catch type errors. C++ is weird!

Reply:
done